### PR TITLE
Updated SRS Compatibility Matrix

### PR DESCRIPTION
**Pull request by bhowd1**
Updated the SRS Compatibility Matrix with a simpler version.

### CODE REVIEW

File: charts/backingservices/charts/srs/README.md [hunk 1]
```diff
@@ -18,8 +18,7 @@ The service deployment provisions runtime service pods along with a dependency o
...
-| \>= 8.6              | \>= 1.20.7  | 7.10.2 or 7.16.3 or 7.17.9     | Pega Infinity 8.6 and later supports using Pega-provided platform-services/search-n-reporting-service Docker Image version 1.20.7 or later with Elasticsearch version, 7.17.9. Pega Infinity 8.8.1 and later requires Docker image 1.20.7 and later. Docker image version 1.20.7 is backward-compatible with Elasticsearch version 7.10.2 and 7.16.3.|
+| \>= 8.6              | 1.20.7  | 7.10.2 or 7.16.3 or 7.17.9      | While SRS Docker images are certified against Elasticsearch versions 7.10.2, 7.16.3 and 7.17.9, Pega recommends using Elasticsearch version 7.17.9. To stay current with Pega releases, use the latest available SRS image 1.20.7.
```
Comment (taz-pega-work) [inline charts/backingservices/charts/srs/README.md:21]: this looks kind of odd>7.10.2 or 7.16.3 or 7.17.9 
Can you make it>>
7.10.2, 7.16.3, and 7.17.9 

The description looks good. ;-)

**Review by taz-pega-work — COMMENTED** [review]
suggesting a slight change to one of the matrix columns

**Review by kingakowalska1 — APPROVED** [review]
Looks good! Thanks!

File: charts/backingservices/charts/srs/README.md (hunk 1 above)
Comment (bhowd1) [reply charts/backingservices/charts/srs/README.md:21]: Sure!! Thanks for your inputs...

Comment (bhowd1) [reply charts/backingservices/charts/srs/README.md]: Done.

**Review by ravitejamekapega — APPROVED** [review]

**Review by chandra-kalavakuri — APPROVED** [review]

**Pull request merged**